src/utils/actions/writeAction.ts

```diff
@@ -68,8 +73,32 @@ const getVotingReputationInstallations = async (
 
 const showActionInActionsList = async (
   colonyAddress: string,
-  initiatorAddress: string,
+  actionFields: ActionFields,
 ): Promise<boolean> => {
+  const { initiatorAddress, recipientAddress, type } = actionFields;
+
+  if (type === ColonyActionType.SetUserRoles) {
```

Comment:
Could we remove this `if` statement and remove the logic below that only targets actions created by the voting rep extn? 

The logic you've added here will also capture that case, meaning we don't need to run it separately.

Reply:
Good catch! I didn't realize this